pkg/apis/core/v1beta1/validation/validation.go

```diff
@@ -17,30 +17,117 @@ limitations under the License.
 package validation
 
 import (
+	"strings"
+
+	apiextv1b1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	apimachineryval "k8s.io/apimachinery/pkg/api/validation"
+	valutil "k8s.io/apimachinery/pkg/util/validation"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
 	"sigs.k8s.io/kubefed/pkg/apis/core/v1beta1"
 )
 
-func ValidateFederatedTypeConfig(object *v1beta1.FederatedTypeConfig) field.ErrorList {
+func ValidateFederatedTypeConfig(obj *v1beta1.FederatedTypeConfig, statusSubResource bool) field.ErrorList {
+
+	allErrs := ValidateFederatedTypeConfigSpec(&obj.Spec, field.NewPath("spec"))
```

Comment:
Is it necessary to validate changes to the spec if the `statusSubResource` is true?